projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -342,6 +360,17 @@ describe('AuthService', () => {
       });
     });
 
+    it('should redirect to the correct route when null is returned', (done) => {
+      (auth0Client.handleRedirectCallback as jasmine.Spy).and.resolveTo(null);
+
+      const localService = createService();
+
+      loaded(localService).subscribe(() => {
+        expect(navigator.navigateByUrl).toHaveBeenCalledWith('/');
+        done();
+      });
+    });
```

Comment:
This test is added to fix coverage. This isnt related to the current PR, but as the coverage was complaining I figured we want to solve this for once and for all.